src/coreclr/src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -8827,15 +8827,15 @@ HRESULT ProfToEEInterfaceImpl::GetReJITIDs(
         // The rejit tables use a lock
         CAN_TAKE_LOCK;
 
-
         PRECONDITION(CheckPointer(pcReJitIds, NULL_OK));
         PRECONDITION(CheckPointer(reJitIds, NULL_OK));
+        PRECONDITION((cReJitIds == 0) == (reJitIds == NULL));
```

Comment:
```suggestion
        PRECONDITION((cReJitIds == 0) || (reJitIds != NULL));
```

Reply:
This changes the semantics though, what I want to express is that either `cReJitIds == 0 && reJitIds == NULL` or `cReJitIds != 0 && reJitIds != NULL`. So I want to make sure the state of both variables matches. With your proposed change it wouldn't trigger for `cReJitIds == 0 && reJitIds != NULL`.